lang/ilias_ru.lang

```diff
@@ -1705,7 +1705,7 @@ chatroom_adm#:#client_settings#:#Client settings
 chatroom_adm#:#enable_osd#:#Enable On-Screen notifications
 chatroom_adm#:#general_settings_title#:#General Chat Settings
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Refreshinterval (seconds)
+chatroom_adm#:#osd_interval#:#Refreshinterval (seconds)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.